draft-pda-protocol.md

```diff
@@ -1253,6 +1257,18 @@ but server implementations may also opt out of participating in a PDA task if
 the minimum batch size is too small. This document does not specify how to
 choose minimum batch sizes.
 
+## Differential privacy {#dp}
```

Comment:
Besides this informal recommendation, do we need explicit protocol support for differential privacy so that collectors can de-noise outputs? We can leave it up to aggregators to decide how they're going to implement DP but I wonder if `PDAOutputShare` should have a field for the epsilon value that was used by the aggregator. Forgive me if I'm talking nonsense about DP, I am speaking in the terms that we used in Prio v2.

Reply:
Hm. I am nervous about being to prescriptive here. In the simplest protocol design nothing is needed since the epsilon is hardcoded into the specific protocol instantiation and won't change.

In practice, some specific instantiations may want to reveal even more information about how noise was applied, e.g.
- The distribution noise is sampled from (Laplace, Gaussian, etc)
- Parameters of the noise distribution
- Any kind of threshold used (for example, if you are using approximate DP)

I think we should make this as opaque to the protocol as possible vs. prescribing some single "epsilon" field which might be too constraining. What do you think?

Reply:
However, I still think this current PR is land-able given that a basic instantiation can hardcode everything without requiring any communication.

Reply:
The way to go here, I think, is to document the open question by adding an `[OPEN ISSUE: blah blah blah]`.

Reply:
Done

Reply:
I put a note about this in #19, which I think is an appropriate issue to track this discussion.